src/main/java/it/gov/pagopa/rtd/ms/rtdmsingestor/model/Transaction.java

```diff
@@ -22,58 +24,88 @@
 public class Transaction {
 
   @CsvBindByPosition(position = 0)
+  @Pattern(regexp = "[a-zA-Z0-9]{5}", message = "ABI length must be 5 alphanumeric char")
   String acquirerCode;
 
   @CsvBindByPosition(position = 1)
+  @Pattern(regexp = "[0-9]{2}", message = "Operation type length must match [0-9]{2}")
   String operationType;
 
   @CsvBindByPosition(position = 2)
+  @Pattern(regexp = "[0-9]{2}", message = "Circuit type length must match [0-9]{2}")
   String circuitType;
 
   @CsvBindByPosition(position = 3)
+  @Pattern(regexp = "[a-zA-Z0-9]{1,64}",
```

Comment:
Is the minimum number of character really 1 here?

Reply:
From the [documentation](https://docs.pagopa.it/digital-transaction-register/v/digital-transaction-filter/acquirer-integration-with-pagopa-centrostella/integration/standard-pagopa-file-transactions) the field is at max 64 alphanumeric char, but it is mandatory (so I avoided the 0 quantity).
I don't have enough information to put a lower limit of char for it.